frontend/templates/views/overview/faq-overview.j2

```diff
@@ -0,0 +1,13 @@
+{% extends '/layouts/default.j2' %}
```

Comment:
As the FAQ overview will only every exist once can we use a similar pattern to `pages/content/amp-dev/documentation/components/index.html` here (using `$view: /views/custom.j2`)?

Reply:
Good point! I'll change it